Readme.md

```diff
@@ -943,7 +943,7 @@ You can configure the help by modifying data properties and methods using `.conf
 
 Simple properties include `sortSubcommands`, `sortOptions`, and `showGlobalOptions`. You can add color using the style methods like `styleTitle()`.
 
-For more detail and examples of changing the displayed text, color, and layout see (./docs/help-in-depth.md).
+For more detail and examples of changing the displayed text, color, and layout see [Help in Depth](./docs/help-in-depth.md).
```

Comment:
Fine in isolation, but for consistency the other links into docs use lower case:

```suggestion
For more detail and examples of changing the displayed text, color, and layout see [help in depth](./docs/help-in-depth.md).
```